docs/getting-started.rst

```diff
@@ -152,6 +168,33 @@ With ``--stop-on-failure``
 
   Result: Failed
 
+Charm mapping
+"""""""""""""
+
+This option enables user to explicitly tell ``juju-verify`` which charm a
+specific application deploys.
+
+By default, ``juju-verify`` parses URL from which the charm was deployed to
```

Comment:
Suggestions to try to have consistency on the wording across sections (running vs deploying, charmstore vs charmhub, and use of passive forms).
```
By default, ``juju-verify`` identifies a charm by parsing the URL that points to the charm bundle.
However, the parser may fail if the charm was deployed from a local source or a non-official charmhub repository.
This option helps specify which charm is used by an application.
```

Reply:
> URL that points to the charm bundle.

I'd avoid using term "charm bundle" as "bundle" has specific meaning within juju/charms.

> However, the parser may fail 

Well, technically, the parser wont fail, just the charm wont be recognized as supported.

I'd suggest following:
```
By default, ``juju-verify`` identifies a charm by parsing the charmhub URL from which it was
deployed. However, the identification of a charm will fail if the charm was deployed from
a local source or a non-official charmhub repository. This option helps specify which charm
is used by an application.
```